package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "@gnosis.pm/safe-singleton-factory",
+  "name": "@safe-global/safe-singleton-factory",
   "version": "1.0.14",
```

Comment:
Is this version expected to be unchanged?

Reply:
This will be the first release of the package republished as safe-global package, and I don't think it needs to be changed as there are no code updates